pytorch_blade/pytorch_blade/quantization/custom_op.cpp

```diff
@@ -65,9 +73,20 @@ torch::Tensor torch_blade_fake_quant(
 }
 
 TORCH_LIBRARY(torch_blade, m) {
-  m.def("fake_quant", torch_blade_fake_quant);
+  m.def("placeholder", placeholder);
+  // By default, the output and the first input of fake_quant will be treated as
+  // alias-pair that shares the same memory space. This will make some jit
+  // passes invalid for fake_quant For example, given a graph like: graph(%self,
+  // %x):
+  //   %1 : Tensor = prim::GetAttr[name="scale"](%self)
+  //   %2 : Tensor = torch_blade::fake_quant(%x, %1, ...)
+  // The prim::GetAttr will not be optimized by ConstantPropagation
+  // which should have been converted to prim::Constant
+  m.def(
```

Comment:
So the schema must be specified to avoid the default behavior？

Reply:
Yes, explicitly tell pytorch that this op will not change the input